src/app/shared/routing/group-route.ts

```diff
@@ -15,22 +17,49 @@ export interface GroupRouteError {
   id?: string;
 }
 
-export function groupRoute(id: GroupId, path: string[]): GroupRoute {
-  return { contentType: 'group', id: id, path };
+
+export type GroupLike =
+  | { id: Group['id'], isUser: boolean }
+  | { id: Group['id'], type: string }
+  | Pick<User, 'groupId' | 'login'>;
+
+function isUser(group: GroupLike): boolean {
+  return ('isUser' in group && group.isUser) || ('login' in group && !!group.login) || ('type' in group && group.type === 'User');
+}
+
+export function rawGroupRoute(group: GroupLike, forceIsUser?: boolean): RawGroupRoute {
```

Comment:
I don't see the point of `forceIsUser` while it can already be forced somehow (with `isUser`) in `GroupLike`.
It is not even used.